### ISSUE TITLE
Default theme "disclose your future plans".

### ISSUE THREAD

**Issue by alexander-gess** [anchorcms/anchor-cms#723]
Came across a funny thing, when you write a lot but you do not publish (expectation of material, not all ready, and etc). We create a few of categories in these categories have a few of drafts. 

But in the public version of the category "disclose your future plans." So, it may seen by users / searches / engins (public). This is my example, sure many anchor bloggers have same:
![empy categories](https://cloud.githubusercontent.com/assets/7859896/4435709/d37cdf36-4753-11e4-84b7-10ece5fce1cb.png)

In other words, the default theme, at the categories list nobody have seen category without public pages.
p.s. thanks for your attention and sorry for my poor English
